velox/exec/tests/MultiFragmentTest.cpp

```diff
@@ -144,11 +172,31 @@ class MultiFragmentTest : public HiveConnectorTestBase,
     task->noMoreSplits("0");
   }
 
-  exec::Split remoteSplit(const std::string& taskId) {
+  static void addHiveSplits(
+      const std::shared_ptr<Task>& task,
+      const std::vector<std::string>& scanNodeIds,
```

Comment:
nit: is it more straightforward to use map from scan node it to a vector hive file paths?